packages/kbn-scout/tsconfig.json

```diff
@@ -27,6 +27,7 @@
     "@kbn/mock-idp-utils",
     "@kbn/test-subj-selector",
     "@kbn/scout-info",
-    "@kbn/scout-reporting"
+    "@kbn/scout-reporting",
+    "@kbn/ftr-common-functional-services"
```

Comment:
I don't like the idea of scout to depend on FTR. @pheyos what do you think?
I wish us to use as much as possible playwright capabilities to retry actions, otherwise port required logic from FTR but without dependencies. We only can pull logic from `kbn/test`